lang/python/python-package.mk

```diff
@@ -138,6 +139,10 @@ define PyBuild/Compile/Default
 	)
 endef
 
+define PyShebang
+$(SED) "1"'!'"b;s,^#"'!'".*python.*,#"'!'"/usr/bin/python2," -i --follow-symlinks $(1)
+endef
```

Comment:
Having shebang handling in multiple places (here and in python-package-install.sh, and also the Py3 version) isn't ideal... Would it be possible to centralize it in one place (either in the shell script or makefiles)?

Reply:
yes it would; and it would a pretty good idea actually;
thanks for pointing it out